runtime/queries/odin/highlights.scm

```diff
@@ -1,141 +1,29 @@
-; Function calls
+(keyword) @keyword
+(operator) @operator
 
-(call_expression
-  function: (identifier) @function)
+(int_literal)   @constant.numeric
+(float_literal) @constant.numeric
+(rune_literal)  @string.character
+(bool_literal) @constant.builtin.boolean
+(nil) @constant.builtin
 
-(call_expression
-  function: (selector_expression
-    field: (field_identifier) @function))
 
+(ERROR) @special
 
```

Comment:
Eventually we want to make the ERROR highlight optional with config since the ERROR node exists for all languages (we could append it based on the config option). For now though we can remove this pattern

Reply:
Perfect, thank you for the help!